### PR TITLE
Changing the twitter logo to X logo #15

### PR DESCRIPTION
**Pull request by tarunkumar2005**
Changing the twitter logo to X logo #15

I have changed the logo of twitter with x logo you can check it and merge this PR and assign this to me and add proper labels, gssoc 2024 ext, hacktoberfest

### CODE REVIEW

**Comment by tarunkumar2005** [comment]
@Mustafiz04 @aqueefcs Please review this PR

**Comment by Mustafiz04** [comment]
@aqueefcs please have a look.

**Comment by aqueefcs** [comment]
@tarunkumar2005 Reviewing you PR. Will update you asap.

**Comment by aqueefcs** [comment]
image

@tarunkumar2005 looks good to me. But currently in the project, in the footer `<svg></svg>` has been used. Can you use fa or fab icons instead. 
You can check package.json file, some icon dependecy is already installed try to use it or feel free to do it in your own way. 
This addition will make the code a lot cleaner. 

Let me if its okay for you.

**Comment by tarunkumar2005** [comment]
I added SVG because svg was added otherwise i could have added the icon from react-icons library if you are fine with that @aqueefcs

**Comment by aqueefcs** [comment]
@tarunkumar2005 Yeah got it. It will be better if we use react-icons. Please feel free to suggest anything.
Can you please update all the icons with react-icons

**Comment by tarunkumar2005** [comment]
@aqueefcs Done bro you can check now put it as level2 now i think as i removed the inconsistency from your codebase

**Comment by aqueefcs** [comment]
can you add the react-icon dependency in package.json.  And there are multiple commits, its best practice to have 1 single commit . Try to squash all the commits into 1

**Comment by tarunkumar2005** [comment]
@aqueefcs I have added the react-icons to package.json i tried to combine all the commits to one but i don't know how to do that so if you can adjust with this or explain me how to do that thing...

**Comment by tarunkumar2005** [comment]
@aqueefcs Can you review and merge it soon its been so many days since it is open.

**Comment by Mustafiz04** [comment]
Code is okay but we are still see 3 commits.

**Comment by tarunkumar2005** [comment]
@Mustafiz04 I don't know how to combine multiple commits into one.

**Comment by tarunkumar2005** [comment]
@Mustafiz04 @aqueefcs Done single commit

**Comment by tarunkumar2005** [comment]
git rebase -i HEAD~5
this command is used 5 is the no of commits you want to check and then change the pick to squash and it is done

**Comment by tarunkumar2005** [comment]
Please review it now.

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1,2 @@
+# These owners will be the default owners for everything in the repo.
+* @GlobeHoppin/admin
```
Comment (Mustafiz04) [inline .github/CODEOWNERS:2]: This is not related to your changes right ?

Comment (tarunkumar2005) [reply .github/CODEOWNERS:2]: no i didn't changed this thing

Comment (Mustafiz04) [reply .github/CODEOWNERS:2]: Yeah this is what happened with your command to squash.
You had 3 commits but you went 5 commits behind.

Follow my steps.

Comment (tarunkumar2005) [reply .github/CODEOWNERS:2]: what to do now chatgpt said this only

Comment (tarunkumar2005) [reply .github/CODEOWNERS:2]: can you once again confirm

Comment (tarunkumar2005) [reply .github/CODEOWNERS:2]: No, squashing does not revert the commits or move you back to a previous state. Instead, squashing combines multiple commits into a single commit without losing the changes those commits made.

**Comment by Mustafiz04** [comment]
Leave it just remove that file and raise PR again, I will approve.

**Pull request closed without merging**